akka-stream/src/main/scala/akka/stream/scaladsl/Compression.scala

```diff
@@ -36,12 +44,22 @@ object Compression {
    * coming out of the flow can be fully decompressed without waiting for additional data. This may
    * come at a compression performance cost for very small chunks.
    *
-   * FIXME: should compression level / strategy / flush mode be configurable? See https://github.com/akka/akka/issues/21849
+   * FIXME: should strategy / flush mode be configurable? See https://github.com/akka/akka/issues/21849
    */
   def deflate: Flow[ByteString, ByteString, NotUsed] =
     CompressionUtils.compressorFlow(() ⇒ new DeflateCompressor)
```

Comment:
Delegate to the method with defaults here to keep call paths simple.

Reply:
:+1: